lib/screens/play/card_viewer.dart

```diff
@@ -42,4 +45,13 @@ class CardViewer extends StatelessWidget {
         ));
   }
 
+  Widget _buildCardNumberWidget(BuildContext context, List<Model.Card> cardList, Model.Card currentCard) {
+    final position = cardList.indexOf(currentCard) + 1;
+    return Container(
+        padding: EdgeInsets.symmetric(vertical: 5),
+        height: MediaQuery.of(context).size.height * 0.03,
```

Comment:
Do we need this? On my emulator it's causing the text to be truncated vertically - it looks fine if I just remove this line.

Incidentally, I see the same thing for the `Resource Count` - again, removing the `height` restriction resolves it (because it lets the `Text` component take exactly the amount of height it needs for its font size). 

![image](https://user-images.githubusercontent.com/5732536/109774613-76319980-7bf8-11eb-84e2-c77fd06f093f.png)

Reply:
I based this off the inclusion of a height setting in the existing _buildCardWidget (or _getCardWidget, as it was originally named). I'm still not totally sure how the layout stuff works. Removing it does look fine for me, but then the tests fail with the error "A RenderFlex overflowed by 40 pixels on the bottom."

Reply:
Which phone are you emulating? I set up a Pixel 4, but I'm thinking I should also use a smaller one.

Reply:
On a Nexus 5, I'm getting all kinds of visual problems:

![image](https://user-images.githubusercontent.com/63930922/110214469-8e294780-7e9c-11eb-8854-9228a137d226.png)

Getting rid of the line you commented on keeps the display looking nice on the Pixel 4 I was using, but I can't figure out how to stop the tests failing without doing something like that. Maybe I'll figure it out after I finish reading the tutorial I linked to in Slack.

Given that display problems exist on other devices anyway, do you think it's ok to merge this for now and address the problems under another PR? Or would you prefer that I keep working at it until I can fix it under this PR?

Reply:
Yeah we do have all kinds of UI funkiness already so I'm happy to merge as is. Only pointed it out as I thought it might be a quick win!